src/lib/browse/InterventionColorSelector.svelte

```diff
@@ -53,6 +30,72 @@
       color
     );
   }
+
+  function styleByFundingProgramme() {
```

Comment:
Could we add `DataDrivenPropertyValueSpecification<string>` as the return here and others? `import type { DataDrivenPropertyValueSpecification } from "maplibre-gl";`